src/spark/utils.py

```diff
@@ -29,11 +32,14 @@ def _get_jars(jar_names: list) -> str:
     return ", ".join(jars)
 
 
-def _get_delta_lake_conf(jars_str: str) -> dict:
+def _get_delta_lake_conf(
+        jars_str: str,
+        executor_cores: int) -> dict:
     """
     Helper function to get Delta Lake specific Spark configuration.
 
     :param jars_str: A comma-separated string of JAR file paths
+    :param executor_cores: The number of CPU cores that each Spark executor will use
```

Comment:
So this only applies if Delta lake is true and the user doesn't call the get base conf method. It seems like it should always apply

Reply:
👍